### ISSUE TITLE
Yet another mismerge

### ISSUE THREAD

**Issue by ddugovic** [ddugovic/Stockfish#576]
God damn it. https://travis-ci.org/github/ddugovic/Stockfish/builds/702976421

test.fen
```
r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R3K2R w KQkq - 0 10 moves e2a6 b4c3
```

build.sh
```
#!/bin/bash
make clean && git checkout origin/master
git log HEAD | grep "\b[Bb]ench[ :]\+[0-9]\{7\}" | head -n 1 | sed "s/[^0-9]*\([0-9]*\).*/\1/g" > git_sig
export benchref=$(cat git_sig)
echo "Reference bench:" $benchref
rm git_sig
make build ARCH=x86-64-bmi2 debug=yes && mv stockfish stockfish_master
make clean && git checkout master

#export CXXFLAGS="-Werror -D_GLIBCXX_DEBUG"
#if [ -x "$(command -v valgrind )" ]; then make clean && make -j3 ARCH=x86-64-bmi2 debug=yes build > /dev/null && ../tests/instrumented.sh --valgrind; fi
make -j3 build ARCH=x86-64-bmi2 COMP=clang debug=yes #&& ../tests/signature.sh $benchref #&& ../tests/perft.sh

#SYZYGY_PATH='/home/lila/Stockfish/tb' ./get_bench.sh bench.log 'all 128 1 13'

./stockfish_master bench 128 1 6819 test.fen nodes |& sed -u -e 's/info depth //' | sed -u -e 's/seldepth //' | sed -u -e 's/multipv //' | sed -u -e 's/tbhits 0 //' | tee 1.txt
./stockfish bench chess 128 1 6819 test.fen nodes |& sed -u -e 's/info depth //' | sed -u -e 's/seldepth //' | sed -u -e 's/multipv //' | sed -u -e 's/tbhits 0 //' | tee 2.txt
sdiff --width=158 1.txt 2.txt
echo "Reference bench:" $benchref

```

**Comment by ddugovic** [comment]
Commenting out most variants seems to help: 64be369

**Comment by ddugovic** [comment]
Fixed by 6cb44a2 as demonstrated by https://travis-ci.org/github/ddugovic/Stockfish/builds/703120588 (the build no longer breaks after a non-functional upstream merge https://travis-ci.org/github/ddugovic/Stockfish/builds/702976421 ).

@ianfab I may need to rebase http://www.variantfishtest.org:6543/tests/view/5ef9bd336e23db104fb88c81 (maybe it requires 6cb44a2 ) so I reduced its game limit to 5000.  Feel free to further reduce that limit and/or stop that test.

**Comment by ianfab** [comment]
@ddugovic If the test is obsolete because the fix does not work without the other one, then please stop it, and otherwise let it finish. However, for such a technical bugfix LTC maybe anyway does not make sense, since the only purpose of testing it is to make sure that the fix works and does not break anything else.
